src/client/opamAction.ml

```diff
@@ -66,10 +66,25 @@ let process_dot_install st nv build_dir =
           OpamFilename.mkdir dst_dir;
         );
         List.iter (fun (base, dst) ->
+            let (base, append) =
+              if exec && not (OpamFilename.exists (OpamFilename.create build_dir base.c)) then
+                let base' =
+                  {base with c = OpamFilename.Base.add_extension base.c "exe"} in
+                if OpamFilename.exists (OpamFilename.create build_dir base'.c) then begin
+                  OpamConsole.warning "Added .exe to %s; .install file should be fixed" (OpamFilename.Base.to_string base.c);
```

Comment:
It's best in general to avoid displaying messages that only make sense to developpers/packagers to the end users, unless they have a clear way to report; but I am not sure how best to do this.

Reply:
Perhaps also noting that the package maintainer should be informed?